client/webserver/webserver.go

```diff
@@ -102,19 +112,28 @@ type clientCore interface {
 
 var _ clientCore = (*core.Core)(nil)
 
+// cachedPassword consists of the seralized crypter and an encrypted password.
+// A key stored in the cookies is used to deserlialize the crypter, then
+// the crypter is used to decrypt the password.
+type cachedPassword struct {
+	EncryptedPass     []byte
+	SerializedCrypter []byte
+}
+
 // WebServer is a single-client http and websocket server enabling a browser
 // interface to the DEX client.
 type WebServer struct {
-	wsServer   *websocket.Server
-	mux        *chi.Mux
-	core       clientCore
-	addr       string
-	csp        string
-	srv        *http.Server
-	html       *templates
-	indent     bool
-	mtx        sync.RWMutex
-	authTokens map[string]bool
+	wsServer        *websocket.Server
+	mux             *chi.Mux
+	core            clientCore
+	addr            string
+	csp             string
+	srv             *http.Server
+	html            *templates
+	indent          bool
+	authMtx         sync.RWMutex
+	authTokens      map[string]bool
+	cachedPasswords map[string]*cachedPassword
```

Comment:
The keys for these maps are the same, right (both auth tokens)?  Don't worry about refactoring at this stage, but it suggests a single map would be adequate if the logic around access to the map were updated to reflect new semantics (`found` would indicate known auth token regardless of value stored including nil, while non-nil value would indicate a cached password).  Rather than change anything, I just request a simple trailing comment on this delc line to doco the map key

```go
cachedPasswords map[string]*cachedPassword // cached passwords keyed by auth token
```

Reply:
Yeah that makes sense, I'll do another PR for this.